diff --git a/node_modules/docco/docco.js b/node_modules/docco/docco.js
index 0bb5a8b..cb3dd38 100644
--- a/node_modules/docco/docco.js
+++ b/node_modules/docco/docco.js
@@ -334,7 +334,7 @@
       // Does the line begin with a comment?
       l.commentMatcher = RegExp(`^\\s*${l.symbol}\\s?`);
       // Ignore [hashbangs](http://en.wikipedia.org/wiki/Shebang_%28Unix%29) and interpolations...
-      l.commentFilter = /(^#![\/]|^\s*#\{)/;
+      l.commentFilter = /(^#![\/]|^\s*#\{|^\s*#.*eslint-.*|^\s*###\sglobal\s.*###$)/;
     }
     return languages;
   };
